src/page/Home/hooks/useRanking.ts

```diff
@@ -1,14 +1,14 @@
 import { requests } from '@/api/requests';
-import type { GiftRankingItem, RankingApiProps } from '@/types';
-import { useQuery } from '@tanstack/react-query';
+import type { ProductInfo, RankingApiProps } from '@/types';
+import { useSuspenseQuery } from '@tanstack/react-query';
 
-const useRanking = ({ activeGenerationButton, activeFilterButton }: RankingApiProps) => {
-  const { data, isLoading } = useQuery<GiftRankingItem[]>({
-    queryKey: ['rankDatas', activeGenerationButton, activeFilterButton],
-    queryFn: () => requests.fetchRanking({ activeGenerationButton, activeFilterButton }),
+const useRanking = ({ activeGeneration, activeFilter }: RankingApiProps) => {
+  const { data } = useSuspenseQuery<ProductInfo[]>({
+    queryKey: ['rankDatas', activeGeneration, activeFilter],
+    queryFn: () => requests.fetchRanking({ activeGeneration, activeFilter }),
   });
 
-  return { rankingDatas: data, isLoading };
+  return { data };
```

Comment:
```suggestion
  return data;
```

이렇게 그냥 값으로 넘겨줘도 무방할 것 같네요!
그러면 사용하는 쪽에서 네이밍을 더 간단하게 할 수 있어요